ios/FluentUI.Demo/FluentUI.Demo/Demos/PeoplePickerDemoController.swift

```diff
@@ -96,4 +96,8 @@ extension PeoplePickerDemoController: PeoplePickerDelegate {
             completion(personas, true)
         }
     }
+
+    func peoplePickerDidHidePersonaSuggestions(_ peoplePicker: PeoplePicker) {
+        UIAccessibility.post(notification: .screenChanged, argument: peoplePicker.badges.last)
```

Comment:
is it a screen change? or more a layout change?

Reply:
Changed to layout change.